nodejs/week2/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const dataAccess = require("../dataAccess.js");
+const router = express.Router();
+const meals = "/data/meals.json";
+const reviews = "/data/reviews.json";
+
+router.get("/cheap-meals", (request, response) => {
```

Comment:
you don't need to register this route with a router just export the function (req, res) => {} and it will be registered via the app.get in index.js. ditto for every router file.